solr/core/src/test/org/apache/solr/core/BlobRepositoryCloudTest.java

```diff
@@ -1,127 +0,0 @@
- * the License.  You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package org.apache.solr.core;
-
-import java.io.IOException;
-import java.nio.ByteBuffer;
-import java.nio.charset.StandardCharsets;
-import java.nio.file.Path;
-import org.apache.solr.client.solrj.SolrQuery;
-import org.apache.solr.client.solrj.SolrServerException;
-import org.apache.solr.client.solrj.impl.CloudSolrClient;
-import org.apache.solr.client.solrj.request.CollectionAdminRequest;
-import org.apache.solr.client.solrj.response.QueryResponse;
-import org.apache.solr.cloud.SolrCloudTestCase;
-import org.apache.solr.common.SolrDocumentList;
-import org.apache.solr.common.SolrInputDocument;
-import org.apache.solr.common.cloud.ZkStateReader;
-import org.apache.solr.common.params.CollectionAdminParams;
-import org.apache.solr.handler.TestBlobHandler;
-import org.junit.BeforeClass;
-import org.junit.Test;
-
-public class BlobRepositoryCloudTest extends SolrCloudTestCase {
-
-  public static final Path TEST_PATH = getFile("solr/configsets").toPath();
-
-  @BeforeClass
-  public static void setupCluster() throws Exception {
-    configureCluster(1) // only sharing *within* a node
-        .addConfig("configname", TEST_PATH.resolve("resource-sharing"))
-        .configure();
-    //    Thread.sleep(2000);
-    CollectionAdminRequest.createCollection(CollectionAdminParams.SYSTEM_COLL, null, 1, 1)
-        .process(cluster.getSolrClient());
-    // test component will fail if it can't find a blob with this data by this name
-    TestBlobHandler.postData(
-        cluster.getSolrClient(),
-        findLiveNodeURI(),
-        "testResource",
-        ByteBuffer.wrap("foo,bar\nbaz,bam".getBytes(StandardCharsets.UTF_8)));
-    //    Thread.sleep(2000);
-    // if these don't load we probably failed to post the blob above
-    CollectionAdminRequest.createCollection("col1", "configname", 1, 1)
-        .process(cluster.getSolrClient());
-    CollectionAdminRequest.createCollection("col2", "configname", 1, 1)
-        .process(cluster.getSolrClient());
-
-    SolrInputDocument document = new SolrInputDocument();
-    document.addField("id", "1");
-    document.addField("text", "col1");
-    CloudSolrClient solrClient = cluster.getSolrClient();
-    solrClient.add("col1", document);
-    solrClient.commit("col1");
-    document = new SolrInputDocument();
-    document.addField("id", "1");
-    document.addField("text", "col2");
-    solrClient.add("col2", document);
-    solrClient.commit("col2");
-    Thread.sleep(2000);
-  }
-
-  @Test
-  public void test() throws Exception {
```

Comment:
This test demonstrates that the object shared remains visible across cores. We need to not loose this. It's really about the memory sharing more than the blobstore.